BitwardenShared/UI/Platform/Application/Support/Localizations/en.lproj/Localizable.strings

```diff
@@ -263,6 +263,7 @@
 "FileSource" = "File Source";
 "FeatureUnavailable" = "Feature unavailable";
 "MaxFileSize" = "Maximum file size is 100 MB.";
+"RequiredMaxFileSize" = "Required. Maximum file size is 100 MB.";
```

Comment:
🎨 Adapt key to what the value is following the pattern in other strings (`MaxFileSize` is not using the right pattern):
```suggestion
"RequiredMaximumFileSizeIsXMB" = "Required. Maximum file size is %1$@ MB.";
```
Also the specific size should be passed as a parameter, otherwise if we change the size it won't reflect on the translations until they're updated.

♻️ Ideally the whole `100 MB` should be passed as parameter with something that humanizes bytes into KB, MB, GB or whatever; but for now I think we're just good with only passing the size and we can revisit this in the future; with a much specialized refactor on these things. I'll create a task to check this.